### PR TITLE
More logger migration

### PR DESCRIPTION
**Pull request by nashif**
Move more drivers/subsystems to the new logger.

### CODE REVIEW

**Review by MaureenHelm — APPROVED** [review]
@carlescufi 	You might want to double check the nrf usb changes.

**Review by carlescufi — APPROVED** [review]
Changes in the USBD driver look fine, not sure why you changed all the tabbing there at the same time though, but it looks fine anyway.

**Comment by nashif** [comment]
> Changes in the USBD driver look fine, not sure why you changed all the tabbing there at the same time though, but it looks fine anyway.

ah, I thought I had this in its own commit, I ran it through uncrustify.

File: drivers/usb/device/usb_dc_nrfx.c [hunk 1]
```diff
@@ -372,7 +372,7 @@ static inline struct usbd_ep_event *usbd_evt_alloc(void)
 			       sizeof(struct usbd_ep_event),
 			       K_NO_WAIT);
 	if (ret < 0) {
-		SYS_LOG_DBG("USBD event alloc failed!");
+		LOG_DBG("USBD event alloc failed!");
```
Comment (finikorg) [inline drivers/usb/device/usb_dc_nrfx.c]: We could use USB_ prefix, otherwise without function name logs are not very useful

**Pull request merged**